floor_timestamp_tests.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+import unittest
+
+class MyTestCase(unittest.TestCase):
```

Comment:
this isn't the right location for tests, can you search for similar tests within the pandas test suite and add it there?

Also, you can use `pytest.mark.parametrize` to parametrize over arguments, no need to write 6 different tests

Reply:
this comment still needs addressing

Reply:
You can disregard floor_timestamp_tests.py. The test cases in pandas/tests/scalar/timestamp/test_timezones.py cover the bug already and they are passing on my end

Reply:
ok - will you remove it?